app/assets/stylesheets/lcms/engine/vendors/foundation_and_overrides.scss

```diff
@@ -1,9 +1,18 @@
+@charset "utf-8";
+
 @import 'foundation';
+
+// If you'd like to include motion-ui the foundation-rails gem comes prepackaged with it, uncomment the 3 @imports, if you are not using the gem you need to install the motion-ui sass package.
+//
 @import 'motion-ui/motion-ui';
 
 // We include everything by default.  To slim your CSS, remove components you don't use.
+
 @include foundation-global-styles;
-@include foundation-flex-grid;
+@include foundation-xy-grid-classes;
```

Comment:
Have some concerns re including `foundation-xy-grid-classes` instead of `foundation-flex-grid`. Apps (this one and hosts too) were created using `foundation-flex-grid` and i don't know how this change will impact, maybe it'll go smoothly.
Good thing that only admin is using assets from engine and ideally it'd be good to get rid of any framework dependency.

Reply:
I reverted the import back to `flex-grid` for now.